src/Sentry/SentryClient.cs

```diff
@@ -168,15 +168,13 @@ private SentryId DoSendEvent(SentryEvent @event, Scope? scope)
                 }
             }
 
-            if (@event.Exception != null && _options.ExceptionFilters?.Count > 0)
```

Comment:
given we have

```
internal List<IExceptionFilter>? ExceptionFilters { get; set; } = new();
```

and it is internal. can we mark is as not nullable?

Reply:
Good catch.  There's some inconsistency about this in other properties on `SentryOptions` as well, so I'll make a separate PR.

Reply:
👍🏻